src/Command/ExportOrdersCommand.php

```diff
@@ -0,0 +1,160 @@
+            case 'parquet':
+                $export = $this->csv2parquet($export);
+                break;
+            case 'csv': break;
+            default:
+                throw new \Exception('Unsupported format');
+        }
+
+        switch ($target) {
+            case 's3':
+                $this->pushToS3(
+                    $export,
+                    $options,
+                    $input->getOption('s3-access-key'),
+                    $input->getOption('s3-secret-key')
+                );
+                break;
+            default:
+                throw new \Exception('Unsupported target');
+        }
+
+
+        return Command::SUCCESS;
+    }
+
+    /**
+     * @param array<mixed> $row
+     * @return array<mixed>
+     */
+    private function formatRow(array $row): array {
+
+        if (count($row) !== 24) {
+            throw new \Exception('Invalid row, expected 24 columns');
+        }
+
+        $__s = fn (string $s): ?string => trim($s) ?: null;
+        $__dt = fn (string $d): ?\DateTime => \DateTime::createFromFormat('d/m/Y H:i:s', $d) ?: null;
+
+        return [
+            'restaurant' => $__s($row[0]),
+            'order_code' => $__s($row[2]),
+            'completed_at' => $__dt($row[4]),
+            'courier' => $__s($row[1]),
+            'fullfillment' => $__s($row[3]),
+            'payment_method' => $__s($row[19]),
+            'delivery_fee' => floatval($row[13]),
+            'tip' => floatval($row[16]),
+            'promotions' => floatval($row[17]),
+            'total_products_excl_vat' => floatval($row[8]),
+            'total_products_incl_vat' => floatval($row[12]),
+            'total_vat' => floatval($row[18]),
+            'stripe_fee' => floatval($row[20]),
+            'platform_fee' => floatval($row[21]),
+            'refunds' => floatval($row[22]),
+            'net_revenue' => floatval($row[23]),
+        ];
+    }
+
+
+    private function csv2parquet(string $csv): string {
+
+        $reader = Reader::createFromString($csv)
+            ->addFormatter(fn($row) => $this->formatRow($row));
+
+        $rows = iterator_to_array($reader);
+        array_shift($rows);
+
+        $schema = Schema::with(
+            FlatColumn::string('restaurant'),
+            FlatColumn::string('order_code'),
+            FlatColumn::dateTime('completed_at'),
+            FlatColumn::string('courier'),
+            FlatColumn::string('fullfillment'),
+            FlatColumn::string('payment_method'),
+            FlatColumn::float('delivery_fee'),
+            FlatColumn::float('tip'),
+            FlatColumn::float('promotions'),
+            FlatColumn::float('total_products_excl_vat'),
+            FlatColumn::float('total_products_incl_vat'),
+            FlatColumn::float('total_vat'),
+            FlatColumn::float('stripe_fee'),
+            FlatColumn::float('platform_fee'),
+            FlatColumn::float('refunds'),
+            FlatColumn::float('net_revenue')
+        );
```

Comment:
It would be preferable to have the exact same column names between CSV and Parquet. This way Parquet and CSV files are interchangeable seamlessly. 
I agree the current column names are messy, but they are known by lots of people, and we can re-map them in Cube with clean names. 
Then, if we want to rename the columns, we rename them in the handler that actually does the export, and the renaming is propagated everywhere.

Reply:
I tried this morning but i got a issue, there is actually duplicate column name, which seems to not being supported by parquet format file
![image](https://github.com/user-attachments/assets/a6e78bec-4b83-4427-9950-49c51b7477c9)